ansible/library/conn_graph_facts.py

```diff
@@ -205,6 +206,37 @@ def _port_alias_to_name(self, device, port):
     def _get_sorted_port_name_list(self, hwsku):
         return natsorted(self._get_port_alias_to_name_map(hwsku).values())
 
+    def _get_port_name_to_alias_map(self, hwsku):
+        """
+        Retrive port name to alias map for specific hwsku.
+        """
+        if hwsku in self._cache_port_name_to_alias:
+            return self._cache_port_name_to_alias[hwsku]
+        port_alias_to_name_map = self._get_port_alias_to_name_map(hwsku)
+        port_name_to_alias_map = dict([(name, alias) for alias, name in port_alias_to_name_map.items()])
+        self._cache_port_name_to_alias[hwsku] = port_name_to_alias_map
+        return port_name_to_alias_map
+
+    def _get_port_name_set(self, device_hostname):
```

Comment:
This function is not used. Remove it?

Reply:
thanks, will remove it.

Reply:
Add it back since need to check whether input interfaces are all names or all aliases, otherwise, raise an exception.